README.md

```diff
@@ -28,17 +28,59 @@ Other than simple transfers from source to destination chains, the token bridge
 ### Dependencies
 
 - [Foundry](https://book.getfoundry.sh/getting-started/installation)
+- [Ginkgo](https://onsi.github.io/ginkgo/#installing-ginkgo) for running the end-to-end tests
 
 ## Structure
 
 - `contracts/` is a Foundry project that includes the implementation of the token bridge contracts
 - `scripts` includes various bash scripts for utility
+- `tests/` includes integration tests for the contracts in `contracts/`, written using the [Ginkgo](https://onsi.github.io/ginkgo/) testing framework.
 
-## Tests
+## Solidity Unit Tests
 
 Unit tests are written under `contracts/test/` and can be run with `forge`:
 
 ```
 cd contracts/test
 forge test -vvv
 ```
+
+## E2E tests
+
+In addition to the docker setup, end-to-end integration tests written using Ginkgo are provided in the `tests/` directory. E2E tests are run as part of CI, but can also be run locally. Any new features or cross-chain example applications checked into the repository should be accompanied by an end-to-end tests.
```

Comment:
We don't have the docker setup in this repo, do we?

Reply:
We use the `teleporter` docker setup through the submodule.

Reply:
I agree with @cam-schultz here, think we can remove references to docker setup.

Reply:
Can you point me to where? I'm not seeing any references in any of the READMEs.

In any case, what I was getting at here is that the phrasing here is a bit awkward. "In addition to" seems to point to a non-existent prior discussion of the Docker setup. We should explicitly point to the `teleporter` submodule setup here.

Reply:
@cam-schultz removed the line